### PR TITLE
New post KB with details how to setup/use a MIDI Control Surface device with Jamulus

### PR DESCRIPTION
**Pull request by henkdegroot**
# Changes
New KB to help user to setup a MIDI device to control the Jamulus Mixerboard.

## Does this need translation?

- [ ] Yes 
- [X] No

### CODE REVIEW

File: _posts/2022-02-01-Midi-Mixerboard-Control.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+This guide explains how to setup your system in order to control the Jamulus Mixerboard using a MIDI Control Surface device (or similar) using JACK.
+The process will be similar for Linux (this also requires JACK and a2jmidid), while for macOS you can use MIDI Studio.
+
+## What do you need?
+1. Jamulus with JACK support (not the standard ASIO version), you can [download the Jamulus JACK version for Windows here]({{ site.download_root_link }}{{ site.download_file_names.windows-jack }})).
+2. You need an audio device with ASIO support or you can use a universal ASIO driver like ASIO4ALL.
+3. You need a MIDI Control Surface. Many different models are available. I am using a Worlde EasyControl 9.
+ ![worlde-easycontrol-9](https://user-images.githubusercontent.com/13550012/152056661-a446f24e-0598-4164-bc0f-a2510ab33c5e.png)
+4. JACK for Winodws [get it from the JACK audio page](https://jackaudio.org/downloads/).
```
Comment (ann0see) [inline _posts/2022-02-01-Midi-Mixerboard-Control.md]: Usually not reviewing KB posts, but there’s a typo (it should be Windows).

Also maybe focus a bit less on Windows. State you‘re using Windows as example, but refer to other OS

File: _posts/2022-02-01-Midi-Mixerboard-Control.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+layout: post
+title: "Control the Jamulus Mixerboard using MIDI"
+lang: "en"
+author: "henkdegroot"
+heading: "How to control the Jamulus Mixerboard using a MIDI Control Surface in Windows using JACK"
+---
+
+This guide explains how to setup your system in order to control the Jamulus Mixerboard using a MIDI Control Surface device (or similar) using JACK.
+The process will be similar for Linux (this also requires JACK and a2jmidid), while for macOS you can use MIDI Studio.
+
```
Comment (ann0see) [inline _posts/2022-02-01-Midi-Mixerboard-Control.md:13]: Add a "More" html comment here. See other posts.

File: _posts/2022-02-01-Midi-Mixerboard-Control.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+You need to know the MIDI Control Change numbers which are assigned to the buttons/faders/knobs of your MIDI Control Surface device. For the device I use, the setup is:
+
+![easycontrol-config](https://user-images.githubusercontent.com/13550012/152056700-9a70b7b0-ea67-4d7f-b56f-a6bc88824394.png)
+
+When your device has buttons which you want to use to control the Mute or Solo option, then you should configure these with the Toggle option (instead of Momentary). In Toggle mode the signal will only be sent once when you press the button and not when you let the button go. In Momentary mode the "ON" signal will be sent when pressed and when let go the "OFF" signal will be sent.
+
+### What to install?
+1. Install JACK for Windows, accepting the default options should be sufficient.
+2. Install Jamulus with JACK for Windows, accepting the default options but don't run Jamulus at the end.
+3. Install ASIO4ALL (if you do not have an audio device with an ASIO driver).
```
Comment (ann0see) [inline _posts/2022-02-01-Midi-Mixerboard-Control.md]: "Install an ASIO Driver" (and don’t stress ASIO4ALL please)

File: _posts/2022-02-01-Midi-Mixerboard-Control.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+lang: "en"
+author: "henkdegroot"
+heading: "How to control the Jamulus Mixerboard using a MIDI Control Surface in Windows using JACK"
+---
+
+This guide explains how to setup your system in order to control the Jamulus Mixerboard using a MIDI Control Surface device (or similar) using JACK.
+The process will be similar for Linux (this also requires JACK and a2jmidid), while for macOS you can use MIDI Studio.
+
+## What do you need?
+1. Jamulus with JACK support (not the standard ASIO version), you can [download the Jamulus JACK version for Windows here]({{ site.download_root_link }}{{ site.download_file_names.windows-jack }})).
```
Comment (ann0see) [inline _posts/2022-02-01-Midi-Mixerboard-Control.md]: Not the standard ASIO version *if you are on Windows*

**Review by ann0see — COMMENTED** [review]
Some comments. Reviewing KB entries is usually not in focus.

BTW, please also update the authors.yml file in _data/ to link your profile

File: _posts/2022-02-01-Midi-Mixerboard-Control.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+We need to add an option to the Target field.
+
+The information you need to provide will depend on the MIDI Control Surface device configuration.
+The mixing section of my device contains 9 faders, 9 knobs and 9 buttons. As I only have 1 button row, I need to decide to use the Mute or Solo option.
+- The fader section starts with CC# 3 and ends with CC# 11.
+- The knobs section starts with CC# 14 and ends with CC# 22.
+- The button sections starts with CC# 23 and ends with CC# 31.
+
+We need to tell Jamulus that these Control Change numbers are being used. When the Control Change is sent, it will also sent a value for the CC. The value will be in the range from 0 till 127.
+The fader at the zero position will sent value 0 and at the max. position it will sent value 127.
```
Comment (ignotus666) [inline _posts/2022-02-01-Midi-Mixerboard-Control.md]: "Send", not "sent".

File: _posts/2022-02-01-Midi-Mixerboard-Control.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+The knob rotate and will sent 0 when turned fully to the left, 64 when in top centre and 127 when turned fully to the right.
```
Comment (ignotus666) [inline _posts/2022-02-01-Midi-Mixerboard-Control.md]: The knob ~~rotate and~~ will **send** 0 when fully turned..."

File: _posts/2022-02-01-Midi-Mixerboard-Control.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+The button will sent OFF (0 value) or ON (127 value).
```
Comment (ignotus666) [inline _posts/2022-02-01-Midi-Mixerboard-Control.md]: send

File: _posts/2022-02-01-Midi-Mixerboard-Control.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+Now let's go back to the JACK Graph dialog window:
+
+![QjackCtl-after-startup-jamulus](https://user-images.githubusercontent.com/13550012/152057213-4c9263a8-f6d7-43e7-91fe-6f40db018b73.png)
+
+In this dialog you should see the Jamulus software which automatically has connected to the System Input (capture) and System Output (playback).
+You notice that there is an extra connect option available for Jamulus, which indicates MIDI.
+
+*NOTE: This is only available when the --ctrlmidich option is used when starting Jamulus.*
+
+Use the mouse and click the "System Midi Capture_1" and drag this to the "Jamulus input midi" connector.
```
Comment (ignotus666) [inline _posts/2022-02-01-Midi-Mixerboard-Control.md]: "...and click ~~the~~ on..."

File: _posts/2022-02-01-Midi-Mixerboard-Control.md (hunk 1 above)
Comment (henkdegroot) [reply _posts/2022-02-01-Midi-Mixerboard-Control.md]: Appreciate you took the time to review this!

File: _posts/2022-02-01-Midi-Mixerboard-Control.md (hunk 2 above)
Comment (henkdegroot) [reply _posts/2022-02-01-Midi-Mixerboard-Control.md:13]: What "More" html....I really can't find any more.

File: _posts/2022-02-01-Midi-Mixerboard-Control.md (hunk 5 above)
Comment (henkdegroot) [reply _posts/2022-02-01-Midi-Mixerboard-Control.md]: Really appreciate you took the time to review this!

File: _posts/2022-02-01-Midi-Mixerboard-Control.md (hunk 7 above)
Comment (henkdegroot) [reply _posts/2022-02-01-Midi-Mixerboard-Control.md]: Seems to be a theme.....with incorrect usage of sent :)

File: _posts/2022-02-01-Midi-Mixerboard-Control.md (hunk 2 above)
Comment (ann0see) [reply _posts/2022-02-01-Midi-Mixerboard-Control.md:13]: This tag:

`<!--more--> `

**Review by ann0see — APPROVED** [review]

**Pull request merged**